sdk/storage/Azure.Storage.Blobs/tests/AppendBlobClientTests.cs

```diff
@@ -1137,6 +1137,40 @@ public async Task AppendBlockAsync_NonZeroStreamPosition()
             TestHelper.AssertSequenceEqual(expectedData, dataResult.ToArray());
         }
 
+        [RecordedTest]
+        [LiveOnly]
+        [ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2022_11_02)]
+        public async Task AppendBlockAsync_HighThroughputAppendBlob()
+        {
+            await using DisposingContainer test = await GetTestContainerAsync();
+
+            // Arrange
+            var blobName = GetNewBlobName();
+            AppendBlobClient blob = InstrumentClient(test.Container.GetAppendBlobClient(blobName));
+            await blob.CreateIfNotExistsAsync();
+            const int blobSize = 5 * Constants.MB;
```

Comment:
```suggestion
            const int oldMaxSize = 4 * Constants.MB;
            const int blobSize = oldMaxSize + Constants.MB;
```
For clarity of chosen size.